src/status_im/contexts/wallet/common/token_value/view.cljs

```diff
@@ -71,47 +73,36 @@
-        bridge-params          (if selected-account?
-                                 {:token            token-data
-                                  :bridge-disabled? (or (not has-balance?)
-                                                        (send-utils/bridge-disabled? token-symbol))
-                                  :stack-id         :screen/wallet.accounts
-                                  :start-flow?      true
-                                  :owners           token-owners}
-                                 {:token-symbol     token-symbol
-                                  :bridge-disabled? (send-utils/bridge-disabled? token-symbol)
-                                  :stack-id         :wallet-stack
-                                  :start-flow?      true
-                                  :owners           token-owners})]
+  (let [token-symbol     (:token token)
+        token-data       (first (rf/sub [:wallet/current-viewing-account-tokens-filtered
+                                         {:query token-symbol}]))
+        selected-account (rf/sub [:wallet/current-viewing-account-address])
+        token-owners     (rf/sub [:wallet/operable-addresses-with-token-symbol token-symbol])
+        params           (merge {:start-flow? true
+                                 :owners      token-owners}
+                                (if selected-account
+                                  {:token        token-data
+                                   :stack-id     :screen/wallet.accounts
+                                   :has-balance? (-> (get-in token [:values :fiat-unformatted-value])
+                                                     (money/greater-than (money/bignumber "0")))}
+                                  {:token-symbol token-symbol
+                                   :stack-id     :wallet-stack}))]
     [quo/action-drawer
      [(cond->> [(when (ff/enabled? ::ff/wallet.assets-modal-manage-tokens)
                   (action-manage-tokens watch-only?))
                 (when (ff/enabled? ::ff/wallet.assets-modal-hide)
                   (action-hide))]
-        (not watch-only?) (concat [(action-buy)
-                                   (when (seq token-owners)
-                                     (action-send send-params entry-point))
-                                   (action-receive selected-account?)
-                                   (when (ff/enabled? ::ff/wallet.swap) (action-swap))
-                                   (when (seq (seq token-owners))
-                                     (action-bridge bridge-params))]))]]))
+        (not watch-only?)
+        (concat [(action-buy)
+                 (when (seq token-owners)
+                   (action-send params entry-point))
+                 (action-receive selected-account)
+                 (when (ff/enabled? ::ff/wallet.swap)
+                   (action-swap params))
+                 (when (seq (seq token-owners))
```

Comment:
I know it already existed, but I wonder if this `(seq (seq x))` makes sense :thinking:

Reply:
Haha, fixed it